nixos/modules/services/networking/syncthing.nix

```diff
@@ -27,12 +42,30 @@ in
         '';
       };
 
+      systemService = mkOption {
+        default = true;
+        description = "Auto launch syncthing as a system service.";
+      };
+
+      useInotify = mkOption {
+        default = false;
+        description = "Watch files using inotify.";
+      };
+
       user = mkOption {
         type = types.string;
         default = defaultUser;
         description = ''
-          Syncthing will be run under this user (user must exist,
-          this can be your user name).
+          Syncthing will be run under this user (user will be created if it doesn't exist.
+          This can be your user name).
+        '';
+      };
+
+      group = mkOption {
+        default = defaultUser;
```

Comment:
Re-using the `defaultUser` name for the group here doesn't feel right. It's a *user*, not a group.

Besides, why does the service even need its own group? Why not use `nogroup`?